Deploy-CAM.ps1

```diff
@@ -2695,23 +2723,17 @@ function Set-RadiusSettings() {
     if ($ignorePrompts -and ($enableRadiusMfa -eq $null)) {
         $enableRadiusMfa = $isRadiusMfaEnabled
     }
-    if ($enableRadiusMfa -eq $null) {
-        $currentRadiusSetting = "disabled"
-        if ($isRadiusMfaEnabled) {
-            $currentRadiusSetting = "enabled"
-        }
-        $changeRadiusSettings = (confirmDialog ([String]::Format("Radius Multi-Factor Authentication is currently {0}. Do you want to change Multi-Factor Authentication settings?", $currentRadiusSetting))) -eq 'y'
-    } else {
-        # Check if any settings are different
-        $changeRadiusSettings = ($enableRadiusMfa -notmatch $isRadiusMfaEnabled) -or `
-                                ($radiusServerHost -and ($radiusServerHost -notmatch $currentRadiusHost)) -or `
-                                ($radiusServerPort -and ($radiusServerPort -notmatch $currentRadiusPort)) -or `
-                                ($radiusSharedSecret)
-    }
 
-    # Update Radius Settings appropriatley
-    if ( $changeRadiusSettings ) {
-        # Load provided Radius Configuration Parameters
+    # Check if any settings are different
+    $changeRadiusSettings = `
+        (($enableRadiusMfa -ne $null)   -and ($enableRadiusMfa -notmatch $isRadiusMfaEnabled)) -or `
+        ($radiusServerHost              -and ($radiusServerHost -notmatch $currentRadiusHost)) -or `
+        ($radiusServerPort              -and ($radiusServerPort -notmatch $currentRadiusPort)) -or `
+        ($radiusSharedSecret)
+
+    # Update RADIUS Settings appropriately
+    if ( $changeRadiusSettings -or (-not $ignorePrompts) ) {
```

Comment:
Right okay so this is saying (since real people don't use ignorePrompts) - if I'm a real person, use this code, if I'm automation only use this code if I'm passing values.

Reply:
Correct, automation should always go to the else block (assuming you're passing values)